### ISSUE TITLE
Missing support for Enum types

### ISSUE THREAD

**Issue by ldorigo** [microsoft/pyright#134]
**Describe the bug**

There seem to be issues with the support for Enum types by Pyright. 

**To Reproduce**
(The following was tested on Python 3.6)

Functional declaration of enums fails:

```python 
from enum import Enum

TEST_ENUM = Enum("TEST_ENUM", "A B C D")

```

(It reports "Expected 1 positional argument")

If I instantiate it as a class, i.e.:

```python

class TEST_ENUM(Enum):
    A = 0
    B = 1
    C = 2
    D = 3
```
It works, but I get the following extra error:

```python
TEST_ERROR["A"]
```

Gives an "Expected no type arguments" error.

**VS Code extension or command-line**

VS Code extension version 1.0.31

**Comment by erictraut** [comment]
Thanks for the bug report. I've added the support, and it will be in the next release.

**Comment by erictraut** [comment]
This is now fixed in version 1.0.33, which I just published. Thanks again for the bug report.
